tox.ini

```diff
@@ -5,14 +5,16 @@ max-line-length = 100
 
 [tox]
 install_command = pip install {opts} {packages}
-envlist = lint,clean,py27-{1.7,1.8,1.9}-{postgres,sqlite},stats
+envlist = lint,clean,py27-{1.7,1.8,1.9}-{postgres,sqlite},py35-{1.8,1.9}-{postgres,sqlite},stats
```

Comment:
Out of curiosity, are you using Django 1.7? I'd like to drop support for that once my work (finally) upgrades, hopefully in the next month or so.